.github/scripts/jpackage.sh

```diff
@@ -0,0 +1,23 @@
+jdeps_modules=$(jdeps --module-path $JAVAFX_HOME --print-module-deps --ignore-missing-deps $GITHUB_WORKSPACE/app/target/lib/SceneBuilder-$VERSION-all.jar)
+JAVAFX_MODULES=javafx.fxml,javafx.media,javafx.swing,javafx.web
+
+$JAVA_HOME/bin/jlink \
+--module-path $JAVAFX_HOME \
+--add-modules $jdeps_modules,$JAVAFX_MODULES \
+--output app/target/runtime \
+--strip-debug --compress 2 --no-header-files --no-man-pages
+
+$JPACKAGE_HOME/bin/jpackage \
+--app-version $VERSION \
```

Comment:
Testing the script, seems `16.0.0-SNAPSHOT` is not accepted as a valid version.

Reply:
Yes, `VERSION` will never contain `-SNAPSHOT`